test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -112,6 +112,7 @@
     ("aten::scatter_reduce.two", datetime.date(2022, 4, 15)),
     ("aten::_s_where", datetime.date(2022, 9, 30)),
     ("aten::quantized_max_pool2d", datetime.date(2022, 4, 15)),
+    ("aten::quantized_max_pool1d", datetime.date(2022, 4, 15)),
```

Comment:
ideally these deprecations should be seaparate PR so that it's easy to revert in case something goes wrong